CMakeLists.txt

```diff
@@ -30,10 +30,6 @@ if(CMAKE_BUILD_TYPE STREQUAL "Release")
 	add_definitions("-DNDEBUG")
 endif()
 
-if (USE_BOOST_REGEX)
-        add_definitions(-DUSE_BOOST_REGEX)
-endif()
-
 option(USE_BOOST_FS "Whether to use boost filsystem instead of c++17" FALSE)
```

Comment:
This option may be deleted too

Reply:
there never was such an option (forgot to add it...)